tests/runtest.py

```diff
@@ -37,6 +37,7 @@
     --interface <interface>     - The interface that TrueNAS is run one
 
 Optional option
+    --markers "smb or nfs"      - Markers to run
```

Comment:
Not easy to indicate the format.  Maybe 
`--markers "<markers to run>"       - Quoted list separated by 'or', e.g. "smb or nfs".   See pytest.ini for list of markers.`